compiler_tester/src/test/case/input/output/mod.rs

```diff
@@ -113,10 +113,10 @@ impl Output {
             .iter()
             .map(|value| {
                 let mut value_str = crate::utils::u256_as_string(value);
-                value_str = value_str.replace(
+                /*value_str = value_str.replace(
```

Comment:
Is this change EVM-only?
I guess it should be branched over target in this case.

Reply:
Yeah, totally. I used the parameter from the callers, passing it around until this function is reached. Maybe there's a better way to achieve this, but it should work.

Reply:
No worries, we need this param there anyway!